plugins/webp-uploads/hooks.php

```diff
@@ -683,16 +695,16 @@ function webp_uploads_wepb_fallback() {
 		s.src = '<?php echo esc_url_raw( plugins_url( '/fallback.js', __FILE__ ) ); ?>';
 
 		i = d.createElement( i );
-		i.src = p + 'jIAAABXRUJQVlA4ICYAAACyAgCdASoCAAEALmk0mk0iIiIiIgBoSygABc6zbAAA/v56QAAAAA==';
+		i.src = p;
 		i.onload = function() {
 			i.onload = undefined;
-			i.src = p + 'h4AAABXRUJQVlA4TBEAAAAvAQAAAAfQ//73v/+BiOh/AAA=';
+			i.src = 'data:image/webp;base64,UklGRh4AAABXRUJQVlA4TBEAAAAvAQAAAAfQ//73v/+BiOh/AAA=';
 		};
 
 		i.onerror = function() {
 			d.body.appendChild( s );
 		};
-	} )( document, 'img', 'script', 'data:image/webp;base64,UklGR' );
+	} )( document, 'img', 'script', '<?php echo esc_html( $detection_string ); ?>' );
 	<?php
 	$javascript = ob_get_clean();
```

Comment:
Here's an alternative which avoids the use of `ob_start()` and ensures that the JS gets IDE syntax checking without needing to mix JS and PHP code: 

```php
	$js_function = <<<JS
		function webpUploadsDetectFallback( d, i, s, p, fallbackSrc ) {
			s = d.createElement( s );
			s.src = fallbackSrc;

			i = d.createElement( i );
			i.src = p;
			i.onload = function() {
				i.onload = undefined;
				i.src = 'data:image/webp;base64,UklGRh4AAABXRUJQVlA4TBEAAAAvAQAAAAfQ//73v/+BiOh/AAA=';
			};

			i.onerror = function() {
				d.body.appendChild( s );
			};
		}
JS;

	wp_print_inline_script_tag(
		sprintf(
			'( %s )( %s, %s )',
			preg_replace( '/\s+/', '', $js_function ),
			wp_json_encode( $detection_string ),
			wp_json_encode( plugins_url( '/fallback.js', __FILE__ ) )
		),
		array(
			'id'            => 'webpUploadsFallbackWebpImages',
			'data-rest-api' => esc_url_raw( trailingslashit( get_rest_url() ) ),
		)
	);
```

In PhpStorm:

Before | After
--|--
![image](https://github.com/WordPress/performance/assets/134745/72996655-d928-4d1e-851d-b73404058052) | ![image](https://github.com/WordPress/performance/assets/134745/c335e3f8-4597-4901-84a1-6c2db3c3723a)

Reply:
sure, i can change this...

Reply:
Updated this as suggested